code/modules/atmospherics/machinery/components/fusion/hfr_core.dm

```diff
@@ -159,6 +159,9 @@
 	///Var used in the meltdown phase
 	var/final_countdown = FALSE
 
+	///Warning damage flags
+	var/warning_damage_flags = NONE
```

Comment:
Sorry but this doc comment is pretty much useless. It should at least mention what the flags are used for, and what are potential flags.